docs/release/trg-0/trg-1-4.md

```diff
@@ -0,0 +1,94 @@
+---
+title: TRG 1.04 - Diagrams as Code
+---
+
+| Status | Created       | Post-History          |
+|--------|---------------|-----------------------|
+| Draft  | 20-Sept-2023  | created initial Draft |
+|        |               |                       |
+
+## Why
+
+To have an easier approach to update current static files and have a better overview of the changes within the product pictures.
```

Comment:
could you please specify that this TRG is about diagrams? Using such broad terms as 'static files' and 'product pictures' could be confusing.

Reply:
i tried to be precise on this with your desired `diagrams`:
https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/401/commits/160bbdf305dc6b3bc0b7d07b1821894ee116634f